### ISSUE TITLE
Can not select customer to a new ticket after canceling the "customer selection" modal dialog

### ISSUE THREAD

**Issue by Tjommel** [osTicket/osTicket#4378]
### Prerequisites

* [X] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [O] Do you have any errors in the PHP error log, or javascript console?
* [x] Did you check the [osTicket forums](http://osticket.com/forum/categories)?
* [X] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/osTicket/osTicket/wiki/Github-Issues-Guidelines)

### Description

If i create a new ticket, a dialog pops up were i can select a customer. if i press "cancel" and after that clicking on the "search button" to asign a customer, the selected customer will not be asignet to the ticket. Instead, the form-fields stay empty.

### Steps to Reproduce

1. Click "New Ticket"
2. "customer selection" window pops up
3. Click "cancel" to close the modal dialog
4. click on the "search" icon near the email edit field
5. Select a customer 
6. The customer is not selected

**Expected behaviour:**
After the second client selection the customer should be assigned to the ticket (step 5)

**Actual behaviour:**
customer is not assigned to ticket and ticket can not be created

**Comment by perryk** [comment]
Yeah, kinda odd.

I can reproduce this issue on a v1.10.4 instance.

If you enter the customer details again after it fails to select it (step 6) it works on the next try.

**Comment by JediKev** [comment]
@perryk 

Unfortunately, I'm unable to replicate this in `1.10.4` nor `1.11.0rc-1`. Below is a GIF of me running through these steps, please let me know if I'm missing something:
![deselect_and_reselect_users](https://user-images.githubusercontent.com/11823401/43402979-8da71428-93d9-11e8-8f23-1952c7e5b822.gif)

Cheers.

**Comment by perryk** [comment]
The GIF shows the exact process I am following also.

I'm using the AD/LDAP module, anyone else using this ?

**Comment by butth34d** [comment]
I don't use LDAP

**Comment by Tjommel** [comment]
tl;dr: Problem was in v1.10.1 and seems to be fixed in v1.10.4.

long version:

Okay i figured out, that this bug is present in v1.10.1 (german version from https://osticket.com.de/), but not in v1.10.4 (normal version from http://osticket.com/download).

I solved this problem by just downloading the current version (v1.10.4) and copied the language files into the i18n folder.

Seems like the problem is on the side of the german comunity (and maybe some more). I recommend to use the original version and use a normal language pack.

Problem solved for me

**Comment by JediKev** [comment]
@Tjommel 

ALWAYS use the official, original version!! We don't encourage using ANY package that's not on osticket.com or here because you don't know what people have added or removed!

Cheers.
